plugins/mongodb.yaml

```diff
@@ -128,15 +128,15 @@ pipeline:
 
   - id: json_parser
     type: json_parser
-    parse_from: $record
+    parse_from: $body
     timestamp:
-      parse_from: $record.t.$date
+      parse_from: $body.t.$date
       #2020-11-03T14:24:07.436-05:00
       layout: '%Y-%m-%dT%H:%M:%S.%f%j'
     severity:
       parse_from: s
       mapping:
-        critical: F
+        fatal: F
```

Comment:
`critical` should map to `error2`

Reply:
Same as my previous comment. This should be fatal.